settings.gradle.kts

```diff
@@ -1,9 +1 @@
 rootProject.name = "kover"
-
-pluginManagement {
-    val kotlinVersion: String by settings
-
-    plugins {
-        kotlin("jvm") version kotlinVersion
-    }
-}
```

Comment:
this is removed because the version is now provided by Gradle's `embeddedKotlinVersion`

Reply:
I think it's better to return the previous version, because Gradle uses old versions.
And if some bug is found in the compiler (for example  https://github.com/Kotlin/kotlinx-kover/blob/a0ff8d9c2c369dd1dbfd43f88e4c0169c262553d/src/main/kotlin/kotlinx/kover/appliers/KoverMergedApplier.kt#L92), then we will have to wait a long time for the release of Gradle, which will use the expected Kotlin.

Reply:
I've reverted it but it does lead to this warning from Gradle

```
> Configure project :
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.5.31` that might work differently than in the requested version `1.7.10`.
```

I've never seen any problems come out of this warning, so I think it can be ignored.

I lean more towards using the same version as Gradle, because at least the compiler bug are known problems. It's never been clear to me what the problems are if the project is compiled with a different version (even if the Kotlin language level is 1.4).